doc/productivity.md

```diff
@@ -93,6 +94,21 @@ To simply check out a commit-ish under a new working directory without disruptin
 git worktree add --checkout ../where-my-checkout-commit-ish-will-live my-checkout-commit-ish
 ```
 
+### Dummy rebases for fixups and execs with `git merge-base`
+
+When rebasing, we often want to do a "dummy rebase," whereby we are not rebasing over an updated master. We can use `git merge-base` to identify the last common commit with master, and rebase off of that.
```

Comment:
Thanks for adding this, always nice to take part of useful productivity tips!
Maybe just me, but could you add something about *why* we "often want to do a dummy rebase"? (What is the context and which specific problem is this solving?)